### DISCUSSION TITLE
Set the order of posts

### DISCUSSION THREAD

**Discussion by lyhsieh** [adityatelange/hugo-PaperMod#765] [Q&A] [answered]
Hello everyone, is it possible to set the order of posts that are displayed on the homepage? Or, can I pin a post at the top of my posts?
![螢幕擷取畫面 (99)](https://user-images.githubusercontent.com/61745234/151110381-6ac4ee7a-96fd-44bc-9783-b83d37debe52.png)
Like the image shown above, or you can directly visit [my site](https://leothebestcoder.github.io/), I want to make "Self Introduction" post always appears on the topmost. Is it possible to do?
Thanks for your help!

**Comment by muhyu** [comment, marked as answer]
https://adityatelange.github.io/hugo-PaperMod/posts/papermod/papermod-faq/#pin-a-post

**Reply by lyhsieh** [reply]
Oh wow!!!
It does work!
Thanks a lot!

Best, Leo
